### ISSUE TITLE
Migration from 2.x to 3.x when tooltip is clipped

### ISSUE THREAD

**Issue by panthony** [sir-dunxalot/ember-tooltips#329]
Trying to migrate from 2.x to 3.x, I'm the following issue:

 If the tooltip's parent has an overflow:hidden (in my case, inside `{{liquid-if}}`), then the tooltip may get clipped.

Previously it was rendered in the wormhole container, now it is rendered directly in parent.

How someone migration from 2.X should fix this point?

Finding & passing manually a wormhole ID does not feel right. Especially since this is an internal details that may change in the future (#319)

**Comment by maxfierke** [comment]
@panthony The [`popperContainer`](https://github.com/sir-dunxalot/ember-tooltips#popper-container) is the escape-hatch you seek. You can use that to control where the tooltip is rendered.

**Comment by panthony** [comment]
Cool!

It's working like before.

Could be worth adding in migration guide though!

**Comment by maxfierke** [comment]
Good idea! Added to Upgrading doc: https://github.com/sir-dunxalot/ember-tooltips/blob/master/UPGRADING-3.x.md#my-tooltips-appear-clipped-use-within-elements-using-overflow-hidden

**Comment by panthony** [comment]
I'll leave this here too in case someone finish here with a similar problem:

I also had a positioning issue with the tooltip in a complex setup (flex layout in table's cell with relative body etc. etc.).

It was occurring only when the scrollable container (which was not the body) was scrolled.

It looked like the scrolling offset was removed instead of added but I failed to find a minimalist reproduction case.

The `popperContainer="body"` also fixed this issue.
